phpsu

```diff
@@ -5,31 +5,40 @@ if (version_compare('7.1.0', PHP_VERSION, '>')) {
     die();
 }
 
+const COMPOSER_INSTALLATION = 0;
+const GIT_INSTALLATION = 1;
+
 call_user_func(function () {
-    $possibleAutoloadFiles = [
+
+    $possibleVendorPaths = [
         // php ./vendor/bin/phpsu
-        __DIR__ . '/../autoload.php',
+        __DIR__ . '/..' => COMPOSER_INSTALLATION,
         // php ./bin/phpsu
-        __DIR__ . '/../vendor/autoload.php',
+        __DIR__ . '/../vendor'  => COMPOSER_INSTALLATION,
         // php ./phpsu
-        __DIR__ . '/vendor/autoload.php',
+        __DIR__ . '/vendor' => GIT_INSTALLATION,
         // php ./vendor/phpsu/phpsu/phpsu may be symlinked from ./vendor/bin/phpsu
-        __DIR__ . '/../../autoload.php',
+        __DIR__ . '/../..' => COMPOSER_INSTALLATION,
     ];
 
-    $firstFileExisting = null;
+    $firstVendorAutoloadFile = null;
 
-    foreach ($possibleAutoloadFiles as $file) {
-        if (file_exists($file)) {
-            $firstFileExisting = $file;
+    foreach ($possibleVendorPaths as $vendorPath => $installationType) {
+        if (file_exists($vendorPath . '/autoload.php')) {
+            define('PHPSU_VENDOR_PATH', $vendorPath);
```

Comment:
didn't we say that we remove these to Constants/Globals?

Reply:
Good catch, I forgot to remove this one